### ISSUE TITLE
Accessibility violation: The tabbable element’s role ‘none’ is not a widget role

### ISSUE THREAD

**Issue by bmamprim** [carbon-design-system/carbon-components-vue#1394]
## Detailed description

> Describe in detail the issue you're having.

This issue is related to CvDataTable. Using IBM Accessibility Checker on the CvDataTable results on the following violation: "The tabbable element's role 'none' is not a widget role"
I have uploaded a print of accessibility checker scan showing the violation and the code.

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

CvDataTable

> What did you expect to happen? What happened instead? What would you like to see changed?

Was expecting no violations. Got an accessibility violation as showed on the screenshot below.

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

"@carbon/vue": "^2.42.0",

## Additional information

- Screenshots or code

<img width="864" alt="Captura de Tela 2023-01-06 às 10 53 26" src="https://user-images.githubusercontent.com/70353626/211026031-89816609-83d5-4e66-a6b1-b1ffeb326514.png">

```html
<div
  :class="`${carbonPrefix}--search-magnifier`"
  ref="magnifier"
>
```

- Notes
